sourcemash/api/categories.py

```diff
@@ -13,18 +22,46 @@
 }
 
 category_status_fields = {
-  'item_count': fields.Integer,
-  'unread_count': fields.Integer
+  'unread': isUnread
 }
 category_status_fields = dict(category_fields, **category_status_fields)
 
+category_status_count_fields = {
+  'item_count': fields.Integer,
```

Comment:
I think we should cut it. It just adds time to API calls without serving enough functionality. It will also make the code much neater.